CONTRIBUTING.md

```diff
@@ -2,59 +2,64 @@
 
-## Filing Issues
-If you'd like to create a new issue, please first make sure the issue does not exist in the [issue list](https://github.com/Expensify/Expensify.cash/issues). When creating a new issue, please include all the required information on the issue template.
-
 ## Payment for Contributions
-We are currently managing payment via Upwork. If you'd like to be paid for your contributions, please apply to fix the issue from our [Upwork issue list](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2). Each issue in this repo will also link out to the associated Upwork job.
-
-## Submitting a Pull Request
-#### Proposing a Change
-1. **Before writing any code**, please post a proposal of the solution you plan to implement as a comment in the GH issue. This should include a brief technical explanation of the changes you will make.
-1. Wait for Expensify to review and provide feedback on the proposal within the GH issue, and assign the GH issue to you. Please do not move forward with creating a Pull Request until your proposal is approved
-1. Fork this repository and create a new branch
-1. ‼️ **Before you start writing any code** ️‼️, please be aware that we require all commits to be [signed](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits). The easiest way to do that is to [generate a new GPG key](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-gpg-key) and [add it to your Github account](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-gpg-key-to-your-github-account). Once you've done that, you can automatically sign all your commits by adding the following to your `.gitconfig`:
-    ```
-    [commit]
-        gpgsign = true
-    [user]
-        email = <Your GH account email>
-        name = <Your Name>
-        signingkey = <your_signing_key>
-    [gpg]
-        program = gpg
-    ```
-1. [Open a PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Be sure to fill in all the required information on the PR template.
-1. An Expensify engineer will be automatically assigned to review your PR
-1. You will need all checks to pass:
-	1. CLA - You must sign our [Contributor License Agreement](https://github.com/Expensify/Expensify.cash/blob/master/CLA.md) by following the CLA bot instructions that will be posted on your PR
-	1. Tests - All tests must pass before a merge of a pull request
-	1. Lint - All code must pass lint checks before a merge of a pull request
-
-#### Testing
-Upon submission of a PR, please include a numbered list of explicit testing steps for each platform (Web, Desktop, iOS, and Android) to confirm the fix works as expected and there are no regressions.
```

Comment:
This fact is missed often and should be explained early.

Reply:
Added back under #### Complete the job